plugins/other/sshd_log

```diff
@@ -28,18 +29,35 @@
 #env.logfile /var/log/messages
 #env.category users
 #
+# config example with journald
+#[sshd_log]
+#group systemd-journal
+#env.logfile journald
+#env.journald_unit sshd.service
```

Comment:
I already hesitated on reusing an env var with a special value: it seems hack-ish on principle, but does work great in practice.
What you propose is in fact a pseudo URL mechanism, we could go full URL, but it's probably overkill. 
BTW journald also has a remote service, on top of HTTP, this doesn't help us.

Let's give it a try !